x/wasm/keeper/bench_test.go

```diff
@@ -56,3 +56,34 @@ func BenchmarkExecution(b *testing.B) {
 		})
 	}
 }
+
+func BenchmarkAPI(b *testing.B) {
+	wasmConfig := types.WasmConfig{MemoryCacheSize: 0}
```

Comment:
I think it's better to add `b.ReportAllocs()` if you want to know the memory usage.